README.md

```diff
@@ -426,6 +411,14 @@ scrollable will default to using the window for the scroll binding.
 
 You can optionally pass an offset value.   This value will be used when calculating if the bottom of the scrollable has been reached.  
 
+* **eventDebounce**
+
+```hbs
+{{infinity-loader eventDebounce=debounceTimer}}
```

Comment:
can we change this to an integer?  Wanna show folks the type that needs to be passed